src/main/java/Duke.java

```diff
@@ -1,10 +1,176 @@
+    public static PrintTextFile printDukeyText = new PrintTextFile("text-art/DukeyText.txt");
+
+    /**
+     * Prints the current list of task when the "list" command is
+     * entered to the terminal.
+     */
+    public static void printTaskList() {
+        System.out.println("    ____________________________________________________________");
+        for (int i = 0; i < numberOfTasks; i += 1) {
+            System.out.print("     ");
+            System.out.print((i + 1) + ".");
+            tasks[i].printStatus();
+        }
+        System.out.println("    ____________________________________________________________");
+    }
+
+    /**
+     * Adds a new todo task with the description of task provided.
+     *
+     * @param userInputString which contains the "todo" command along with
+     *                        the description of the todo task. No time
+     *                        details is needed here.
+     */
+    public static void addToDo(String userInputString) {
+        String taskName = userInputString.substring(5);
+
+        tasks[numberOfTasks] = new ToDo(taskName);
+        numberOfTasks += 1;
+
+        tasks[numberOfTasks - 1].printAddingStatus(numberOfTasks - 1);
+    }
+
+    /**
+     * Adds a new deadline task with the description of deadline task provided
+     * along with the time which is due "by".
+     *
+     * @param userInputString which contains the "deadline" command along with
+     *                        the description of the deadline task and followed
+     *                        by a "/" to separate the due time "by".
+     */
+    public static void addDeadline(String userInputString) {
+        String taskName = userInputString.substring(9).split("/")[0];
+        String by = userInputString.substring(9).split("/")[1];
+
+        tasks[numberOfTasks] = new Deadline(taskName, by);
+        numberOfTasks += 1;
+
+        tasks[numberOfTasks - 1].printAddingStatus(numberOfTasks - 1);
+    }
+
+    /**
+     * Adds a new event task with the description of task provided and time
+     * which it is happening "at".
+     *
+     * @param userInputString which contains the "event" command along with
+     *                        the description of the event task followed by
+     *                        a "/" to separate the happening time "at".
+     */
+    public static void addEvent(String userInputString) {
+        String taskName = userInputString.substring(7).split("/")[0];
+        String by = userInputString.substring(7).split("/")[1];
+
+        tasks[numberOfTasks] = new Event(taskName, by);
+        numberOfTasks += 1;
+
+        tasks[numberOfTasks - 1].printAddingStatus(numberOfTasks - 1);
+    }
+
+    /**
+     * Marks a certain task based on task number as finished.
+     *
+     * @param userInputString which contains the "done" command along with the
+     *                        finished task number.
+     */
+    public static void finishTask(String userInputString) {
```

Comment:
Names of functions were clear, I was able to understand what each function does just from the name.